controllers/authController.js

```diff
@@ -48,6 +49,8 @@ exports.register = async (req, res) => {
         message: 'User already exists!'
       })
     }
+    
+    sendVerifMail(req.body.email).catch(console.error);
```

Comment:
We have to send the email right before to respond back to the client.

Reply:
Also, we don't send a verify email, just a welcome email.
So the function should be call `sendEmail`.